_templates/react-native-typescript-component/new/export.ejs

```diff
@@ -1,4 +1,5 @@
 ---
 to: <%= root %>/<%= path %>/<%= h.inflection.transform(name, ['undasherize', 'classify']) %>/index.ts
 ---
-export * from './<%= h.inflection.transform(name, ['undasherize', 'classify']) %>';
\ No newline at end of file
+export * from './<%= h.inflection.transform(name, ['undasherize', 'classify'])
```

Comment:
```suggestion
export * from './<%= h.inflection.transform(name, ['undasherize', 'classify']) %>'
```
Guessing your linter/prettier did this but this should stay on the same line

Reply:
Yes, thanks linter...I'll fix